core/ledger/kvledger/txmgmt/statedb/statecouchdb/statecouchdb_test_export.go

```diff
@@ -16,9 +16,11 @@ import (
 	"github.com/hyperledger/fabric/common/metrics/disabled"
 	"github.com/hyperledger/fabric/core/ledger/kvledger/txmgmt/statedb"
 	"github.com/hyperledger/fabric/core/ledger/util/couchdb"
+	"github.com/hyperledger/fabric/core/ledger/util/couchdbtest"
```

Comment:
The new addition of the code in this file is not correct. The consumers of "TestVDBEnv" defined in this file, start couch and stop couchdb explicitly. I would suggest to review the code of the consumers and if you want to move the code for starting/stopping here, better to enclose this in the TestVDBEnv but still controlling the start and stop explicitly in the consumer so that it can be done once at the package level...